gateway/ln-gateway/src/lib.rs

```diff
@@ -277,7 +277,9 @@ impl Gateway {
                 .expect("Failed to build gateway client"),
         );
 
-        if let Err(e) = self.load_actor(client.clone(), route_hints).await {
+        let actor = self.load_actor(client.clone(), route_hints).await;
```

Comment:
I'm not sure if would be a good pattern/standard (@dpc  might correct me here 😁) but you could already get the actor, and info all in a single let federation_info.

Reply:
thanks, followed up with a fixup